### PR TITLE
Partner Page (only missing one uni-org)

### PR DESCRIPTION
**Pull request by Raitou**
As I can't find info about the partner_4 in the uni org it won't be there. In the meantime until information about partner_4 has been found this will be the current final partner page.

### CODE REVIEW

**Comment by Simperfy** [comment]
Images are not found on deployment, please prefix all image path with "/"
image

File: src/components/Dictionaries/PartnerStrings.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    "placeholder_long" : "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc interdum sollicitudin purus, eu sollicitudin nisi porta in. Integer fringilla aliquet felis, et convallis ex euismod nec.",
+    "page_partners" : "Our Partners",
+    "desc_partners" : "We at CS EXPO 2022 have partnered with exceptional companies and organizations in order to bring this event to fruition. Check them out and get to know them.",
+};
+
+const partnerStrGB = {
+    "Partners_Title" : "ALSO BROUGHT TO YOU BY",
+
+    "Partner_1_Title" : "START UP PODCAST",
+    "Partner_1_Image" : "images/partners/gigabyte/partner_1.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:13]:
```suggestion
    "Partner_1_Image" : "/images/partners/gigabyte/partner_1.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_1_Link_2_Description" : "YouTube",
+    "Partner_1_Link_3" : "https://podcasts.apple.com/us/podcast/start-up-podcast/id1576462394",
+    "Partner_1_Link_3_Description" : "Apple Podcast",
+    "Partner_1_Link_4" : "https://open.spotify.com/show/6BObuPvMfoZzdlJeb1XXVa",
+    "Partner_1_Link_4_Description" : "Spotify Podcast",
+    "Partner_1_Email" : "mailto:whatsup@startuppodcast.club",
+    "Partner_1_Email_Description" : "whatsup@startuppodcast.club",
+
+    "Partner_2_Title" : "CYBER OPTIMUS PHILIPPINES, INC.",
+    "Partner_2_Image" : "images/partners/gigabyte/partner_2.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:26]:
```suggestion
    "Partner_2_Image" : "/images/partners/gigabyte/partner_2.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_2_Link_1" : "https://www.facebook.com/cyberoptimusph/",
+    "Partner_2_Link_1_Description" : "Facebook",
+    "Partner_2_Link_2" : "https://www.cyberoptimus.com/",
+    "Partner_2_Link_2_Description" : "Website",
+    "Partner_2_Email" : "mailto:contact.us@cyberoptimus.com",
+    "Partner_2_Email_Description" : "contact.us@cyberoptimus.com",
+
+    "Partner_3_Title" : "TRASH CASH PH",
+    "Partner_3_Image" : "images/partners/gigabyte/partner_3.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:35]:
```suggestion
    "Partner_3_Image" : "/images/partners/gigabyte/partner_3.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_3_Link_1" : "https://www.facebook.com/trashcashph/",
+    "Partner_3_Link_1_Description" : "Facebook",
+    "Partner_3_Link_2" : "https://trashcash.ph/",
+    "Partner_3_Link_2_Description" : "Website",
+    "Partner_3_Email" : "mailto:hello@trashcash.ph",
+    "Partner_3_Email_Description" : "hello@trashcash.ph",
+
+    "Partner_4_Title" : "8CHAIN",
+    "Partner_4_Image" : "images/partners/gigabyte/partner_4.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:44]:
```suggestion
    "Partner_4_Image" : "/images/partners/gigabyte/partner_4.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_4_Link_1" : "https://www.facebook.com/8CHAIN/",
+    "Partner_4_Link_1_Description" : "Facebook",
+    "Partner_4_Link_2" : "https://www.8chain.io/",
+    "Partner_4_Link_2_Description" : "Website",
+    "Partner_4_Email" : "mailto:partnership@8chain.io",
+    "Partner_4_Email_Description" : "partnership@8chain.io",
+    
+    "Partner_5_Title" : "WICODE FOR KIDS",
+    "Partner_5_Image" : "images/partners/gigabyte/partner_5.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:53]:
```suggestion
    "Partner_5_Image" : "/images/partners/gigabyte/partner_5.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_5_Link_1" : "https://www.facebook.com/wicodeforkids/",
+    "Partner_5_Link_1_Description" : "Facebook",
+    "Partner_5_Email" : "mailto:batangaschapter@witech.org",
+    "Partner_5_Email_Description" : "batangaschapter@witech.org",
+
+    "Partner_6_Title" : "PROSPERNA",
+    "Partner_6_Image" : "images/partners/gigabyte/partner_6.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:60]:
```suggestion
    "Partner_6_Image" : "/images/partners/gigabyte/partner_6.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_6_Link_1" : "https://www.facebook.com/prosperna/",
+    "Partner_6_Link_1_Description" : "Facebook",
+    "Partner_6_Link_2" : "https://prosperna.com/",
+    "Partner_6_Link_2_Description" : "Website",
+    "Partner_6_Email" : "mailto:support@prosperna.com",
+    "Partner_6_Email_Description" : "support@prosperna.com",
+
+    "Partner_7_Title" : "ORANGE & BRONZE SOFTWARE LABS",
+    "Partner_7_Image" : "images/partners/gigabyte/partner_7.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:69]:
```suggestion
    "Partner_7_Image" : "/images/partners/gigabyte/partner_7.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_7_Link_1" : "https://www.facebook.com/orangeandbronze",
+    "Partner_7_Link_1_Description" : "Facebook",
+    "Partner_7_Link_2" : "http://www.orangeandbronze.com/",
+    "Partner_7_Link_2_Description" : "Website",
+    "Partner_7_Email" : "mailto:info@orangeandbronze.com",
+    "Partner_7_Email_Description" : "info@orangeandbronze.com",
+    
+    "Partner_8_Title" : "PINOY IP WORKS, INCORPORATED",
+    "Partner_8_Image" : "images/partners/gigabyte/partner_8.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:78]:
```suggestion
    "Partner_8_Image" : "/images/partners/gigabyte/partner_8.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+    "placeholder_long" : "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc interdum sollicitudin purus, eu sollicitudin nisi porta in. Integer fringilla aliquet felis, et convallis ex euismod nec.",
+    "page_partners" : "Our Partners",
+    "desc_partners" : "We at CS EXPO 2022 have partnered with exceptional companies and organizations in order to bring this event to fruition. Check them out and get to know them.",
+};
+
+const partnerUni = {
+    "Partners_Title" : "UNIVERSITY ORG PARTNERS",
+
+    "Partner_1_Title" : "UP DILIMAN ACM STUDENT CHAPTER",
+    "Partner_1_Image" : "images/partners/uni/partner_1.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:114]:
```suggestion
    "Partner_1_Image" : "/images/partners/uni/partner_1.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_1_Link_1_Description" : "Facebook",
+    "Partner_1_Link_2" : "https://twitter.com/upacm",
+    "Partner_1_Link_2_Description" : "Twitter",
+    "Partner_1_Link_3" : "https://upacm.net/",
+    "Partner_1_Link_3_Description" : "Website",
+    "Partner_1_Email" : "mailto:council@upacm.net",
+    "Partner_1_Email_Description" : "council@upacm.net",
+
+    "Partner_2_Title" : "FEU-TECH ACM STUDENT CHAPTER",
+    "Partner_2_Image" : "images/partners/uni/partner_2.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:125]:
```suggestion
    "Partner_2_Image" : "/images/partners/uni/partner_2.jpg",
```

File: src/components/Dictionaries/PartnerStrings.js [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+    "Partner_2_Link_1_Description" : "Facebook",
+    "Partner_2_Link_2" : "https://twitter.com/feutechACM",
+    "Partner_2_Link_2_Description" : "Twitter",
+    "Partner_2_Link_3" : "https://feutech.acm.org/",
+    "Partner_2_Link_3_Description" : "Website",
+    "Partner_2_Email" : "mailto:acm.feu.it@gmail.com",
+    "Partner_2_Email_Description" : "acm.feu.it@gmail.com",
+
+    "Partner_3_Title" : "NU COMPUTER SOCIETY",
+    "Partner_3_Image" : "images/partners/uni/partner_3.jpg",
```
Comment (Simperfy) [inline src/components/Dictionaries/PartnerStrings.js:136]:
```suggestion
    "Partner_3_Image" : "/images/partners/uni/partner_3.jpg",
```

**Review by Simperfy — CHANGES_REQUESTED** [review]

**Comment by Simperfy** [comment]
I suggest adding spacing to the top of image and lessen the gap with the social links.
image

**Comment by DauntlessDev** [comment]
Great Suggestions @Simperfy

**Comment by Raitou** [comment]
Please check latest pull regarding this for the update thanks!

**Pull request closed without merging**